packages/core-react/index.d.ts

```diff
@@ -37,9 +37,19 @@ export type ButtonProps = {
 
 declare const Button: React.FC<ButtonProps>;
 
+export type CheckboxProps = {
+  className?: string,
+  id?: string,
+  label?: React.ReactNode
+  name?: string,
```

Comment:
What is `name` for again?

Reply:
![Screen Shot 2019-09-05 at 1 21 12 PM](https://user-images.githubusercontent.com/45638740/64364087-2d3b5d80-cfe0-11e9-8cb3-8200dcb12cf3.png)
 for the checkbox, I based on the props that I see in index file for checkbox